tests/unit/validators/date-test.js

```diff
@@ -0,0 +1,265 @@
+import moment from 'moment';
```

Comment:
I used it in the unit tests since the dependency was needed anyway so might as well use the closest thing possible to the validator implementation. If ember-validators does remove `moment` it would indeed be better to work without it in tests.

Reply:
For the extremely simple cases ember-validators uses it for, it definitely should be optional and use the platform if so.  This test would still be useful as the API would look like:

`validator(key, 'Nov || 3rd || 1998', { dateFn: moment })` or something like that